torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -170,7 +173,17 @@ def _find_the_perfect_or_nearest_match_onnxfunction(
         for symbolic_function in reversed(default_and_custom_functions):
             overload_func = symbolic_function.onnx_function
             function_opschema = _OpSchemaWrapper(overload_func.op_schema)
-            if function_opschema.perfect_match_inputs(onnx_args, onnx_kwargs):
+
+            # NOTE: OnnxFunction does not have the same function signature as the original
+            # PyTorch operator. We need to separate the input attributes from the arguments.
+            function_args, function_kwargs = _separate_input_attributes_from_arguments(
```

Comment:
It would be a little weird, If I directly put this function inside `_OpSchemaWrapper`. So I changed the input to OnnxFunction. This way it is kind of extended to both schema usages. The function right now is called inside `find_perfect_match`.